app/preload.js

```diff
@@ -14,7 +14,9 @@ ipcRenderer.on('notification', (_, metadata) => showNotification(metadata))
 
 function sendTrackMetadata(sender) {
   const artworkURL = getArtworkURL()
-  sender.send('trackMetadata', { artworkURL })
+  const likeStatus = getLikeStatus()
+  const trackMetadata = { artworkURL: artworkURL, isLiked: likeStatus }
+  sender.send('trackMetadata', { trackMetadata })
```

Comment:
FYI `const foo = 'bar'; const baz = { foo }` results in the variable `baz` having a value `{ foo: 'bar' }` therefore what we should have here is something like this:

```js
const artworkURL = getArtworkURL()
const isLiked = getLikeStatus()
sender.send('trackMetadata', { artworkURL, isLiked })
```